packages/create-waku/package.json

```diff
@@ -23,17 +23,19 @@
     "dev": "pnpm build --watch",
     "compile": "rm -rf template dist *.tsbuildinfo && pnpm run template && pnpm run build",
     "template": "cp -r ../../examples template && rm -rf template/*/dist && rm -rf template/*/node_modules && rm -rf template/*.tsbuildinfo && (for d in template/*; do mv $d/.gitignore $d/gitignore || true; done)",
-    "build": "tsup"
+    "build": "tsup",
+    "test": "vitest run"
   },
   "devDependencies": {
+    "@clack/prompts": "^0.10.0",
```

Comment:
It may not matter much, but let's move dependencies that are directly used from `./src` to `"dependencies"`.

Reply:
This makes it so the deps are installed by users instead of written into the bundle. Sounds good to me though 👍 - I have no preference.

Reply:
> This makes it so the deps are installed by users instead of written into the bundle.

What? I thought there were no differences in our case.

Reply:
No, so this is the top of the bundled file when they're `dependencies`

<img width="974" alt="image" src="https://github.com/user-attachments/assets/b3a9992e-76ad-4081-a65f-9f56a7078946" />

When they're `devDependencies` the bundler takes the functions we use from `devDependencies` and bundles these into the `dist/index.js`. 

So when using `dependencies`:
- deps are imported to dist/index.js
- deps are installed to node_modules by users

When using `devDependencies`:
= deps are bundled into dist/index.js
- users have no need to install dependencies

This is expected behavior with the bundler as well. The devDeps approach does make for less moving pieces, but it should all be abstracted away from users.

Reply:
Then, I guess having everything in `devDependenceis` was intentional. Sorry. Let's revert 98dd8e8bcf0a00ea7710fef033ab6d717fabbaf5.